hack/build/operator/build

```diff
@@ -4,21 +4,30 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if ! which docker > /dev/null; then
-	echo "docker needs to be installed"
+if ! which go > /dev/null; then
+	echo "golang needs to be installed"
 	exit 1
 fi
 
-if ! which gcloud > /dev/null; then
+if ! which docker > /dev/null; then
 	echo "docker needs to be installed"
 	exit 1
 fi
 
 : ${IMAGE:?"Need to set IMAGE, e.g. gcr.io/coreos-k8s-scale-testing/etcd-operator"}
 
+mkdir -p _output/bin/ || true
```

Comment:
> why not just /bin?

We have "_output" in .gitignore. We should probably use temp dir. Not bother to do it here.

Using a variable like $output_dir sounds reasonable